data/config-schema/src/main/resources/net/corda/schema/configuration/db/1.0/corda.db.json

```diff
@@ -4,5 +4,56 @@
   "title": "Corda Database Configuration Schema",
   "description": "Configuration schema for the database subsection.",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "database": {
+      "description": "Database settings",
+      "type": "object",
+      "default": {},
+      "properties": {
+        "user": {
+          "description": "Database user",
+          "type": "string"
+        },
+        "pass": {
+          "description": "Database password",
+          "type": "string"
+        },
+        "jdbc": {
+          "description": "JDBC settings",
+          "type": "object",
+          "default": {},
+          "properties": {
+            "driver": {
+              "description": "JDBC driver",
+              "type": "string"
```

Comment:
I think we should have a default for this.

Reply:
set default value to "org.postgresql.Driver"